transportable-udfs-plugin/src/main/java/com/linkedin/transport/plugin/Defaults.java

```diff
@@ -0,0 +1,102 @@
+/**
+ * Copyright 2019 LinkedIn Corporation. All rights reserved.
+ * Licensed under the BSD-2 Clause license.
+ * See LICENSE in the project root for license information.
+ */
+package com.linkedin.transport.plugin;
+
+import com.google.common.collect.ImmutableList;
+import com.linkedin.transport.codegen.HiveWrapperGenerator;
+import com.linkedin.transport.codegen.PrestoWrapperGenerator;
+import com.linkedin.transport.codegen.SparkWrapperGenerator;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.List;
+import java.util.Properties;
+
+import static com.linkedin.transport.plugin.ConfigurationType.*;
+
+
+/**
+ * Stores default configurations for the Transport UDF plugin
+ */
+class Defaults {
+
+  private Defaults() {
+  }
+
+  // The versions of the Transport and supported platforms to apply corresponding versions of the platform dependencies
+  private static final Properties DEFAULT_VERSIONS = loadDefaultVersions();
+
+  private static Properties loadDefaultVersions() {
+    try (InputStream is =
+        Thread.currentThread().getContextClassLoader().getResourceAsStream("version-info.properties")) {
+      Properties defaultVersions = new Properties();
+      defaultVersions.load(is);
+      return defaultVersions;
+    } catch (IOException e) {
+      throw new RuntimeException("Error loading version-info.properties", e);
+    }
+  }
+
+  static final List<DependencyConfiguration> MAIN_SOURCE_SET_DEPENDENCIES = ImmutableList.of(
```

Comment:
DEPDENCY_CONFIGURATIONS

Reply:
Updated and should be fixed everywhere now. Sorry about that!